sormas-backend/src/main/java/de/symeda/sormas/backend/task/TaskJurisdictionPredicateValidator.java

```diff
@@ -0,0 +1,65 @@
+package de.symeda.sormas.backend.task;
+
+import javax.persistence.criteria.CriteriaBuilder;
+import javax.persistence.criteria.Predicate;
+
+import de.symeda.sormas.api.utils.jurisdiction.JurisdictionValidator;
+import de.symeda.sormas.backend.user.User;
+
+public class TaskJurisdictionPredicateValidator extends JurisdictionValidator<Predicate> {
```

Comment:
Reading this class I have the impression that task are never in the jurisdiction of the user. I miss the check for jurisdiction of associated object.

The JurisdictionValidator and it's implementations need a bit of refactoring

Reply:
have refactored the jurisdiction validators